src/main/java/com/applitools/imagetester/lib/EyesFactory.java

```diff
@@ -15,211 +15,207 @@
+		this.matchLevel = ml;
+		return this;
+	}
+
+	public EyesFactory proxySettings(ProxySettings proxySettings) {
+		this.proxySettings = proxySettings;
+		return this;
+	}
+
+	public EyesFactory branch(String branch) {
+		this.branch = branch;
+		return this;
+	}
+
+	public EyesFactory parentBranch(String parentBranch) {
+		this.parentBranch = parentBranch;
+		return this;
+	}
+
+	public EyesFactory baselineEnvName(String envName) {
+		this.envName = envName;
+		return this;
+	}
+
+	public EyesFactory logFile(String filename) {
+		if (filename != null)
+			this.logger.printMessage("------------------------------------------ \n"
+					+ "Setting the log file path using the 'lf' flag has been DEPRECATED. \n"
+					+ "The logs are automatically created and saved to: \n" + "Mac/Linux - $TMPDIR/applitools-logs/ \n"
+					+ "Windows (PowerShell) - $env:TEMP/applitools-logs/ \n"
+					+ "The log directory can now be specified using this environment variable: \n"
+					+ "APPLITOOLS_LOG_DIR=<path> \n" + "------------------------------------------ \n");
+		return this;
+	}
+
+	public EyesFactory hostOs(String os) {
+		this.hostOs = os;
+		return this;
+	}
+
+	public EyesFactory hostApp(String app) {
+		this.hostApp = app;
+		return this;
+	}
+
+	public EyesFactory environmentName(String environmentName) {
+		this.environmentName = environmentName;
+		return this;
+	}
+
+	public EyesFactory saveFaliedTests(boolean saveFailed) {
+		this.saveFailed = saveFailed;
+		return this;
+	}
+
+	public EyesFactory ignoreDisplacement(boolean ignore) {
+		this.ignoreDisplacement = ignore;
+		return this;
+	}
+
+	public EyesFactory saveNewTests(boolean save) {
+		this.saveNewTests = save;
+		return this;
+	}
+
+	public EyesFactory imageCut(String[] cutArgs) {
+		if (cutArgs == null)
+			return this;
+		if (cutArgs.length > 4)
+			throw new IllegalArgumentException(
+					"Image cut must contain at most four parameters (header,footer,left,right)");
+		this.cutValues = Arrays.stream(cutArgs).mapToInt((s) -> StringUtils.isNumeric(s) ? Integer.parseInt(s) : 0)
+				.toArray();
+		return this;
+	}
+
+	public EyesFactory accSettings(String[] settings) {
+		if (settings == null)
+			return this;
+		AccessibilityLevel level = AccessibilityLevel.AA;
+		AccessibilityGuidelinesVersion version = AccessibilityGuidelinesVersion.WCAG_2_0;
+		if (settings.length > 2)
+			throw new IllegalArgumentException("incorrect Accessibility setting");
+		if (settings.length > 0) {
+			level = StringUtils.isNotBlank(settings[0]) ? Utils.parseEnum(AccessibilityLevel.class, settings[0], "_")
+					: level;
+			version = settings.length == 2 && StringUtils.isNotBlank(settings[1])
+					? Utils.parseEnum(AccessibilityGuidelinesVersion.class, settings[1], "_")
+					: version;
+		}
+		this.accSettings = new AccessibilitySettings(level, version);
+		return this;
+	}
+
+	public EyesFactory logHandler(boolean logHandler) {
+		this.logHandler = logHandler;
+		return this;
```

Comment:
The contents of EyesFactory got tabbed over by one. Please fix that spacing, and this LGTM!

Reply:
How does it look now?